src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -78,6 +78,12 @@ func validateConfig(_cfg yamlconf.Config) error {
 	if !ok {
 		return fmt.Errorf("Config is not a flashlight config!")
 	}
+	if cfg.Client == nil {
+		return fmt.Errorf("No client config")
+	}
+	if cfg.Client.ChainedServers == nil {
+		return fmt.Errorf("No chained servers")
+	}
```

Comment:
Yeah just removed it @oxtoacart -- I think that client-reported error isn't possible with what's on devel actually!